src/domain/graphql/venue/venueSchema.ts

```diff
@@ -62,6 +67,7 @@ const typeDefs = gql`
     isOpen: Boolean
     ontologyTree: [Ontology]!
     ontologyWords: [Ontology]!
+    accessibilitySentences: [AccessibilitySentences]
```

Comment:
I think that the array can be required here (`!`) as you are using an empty array as a default value in `src/domain/graphql/venue/venueResolver.ts`.